README.md

```diff
@@ -1,47 +1,47 @@
-# Fennec
+# MongooseICE
 
 [![Build Status][BUILD BADGE]][BUILD LINK]
 [![Coverage Status][COVERAGE BADGE]][COVERAGE LINK]
 
-[Documentation](https://hexdocs.pm/fennec/0.3.0)
+[Documentation](https://hexdocs.pm/MongooseICE/0.3.0)
```

Comment:
Package names on hex can't contain capital letters - it should be changed to `mongooseice`.